plugin/src/shared/utils/cli-utils.ts

```diff
@@ -118,4 +118,33 @@ export const selectApplication = async (tree: Tree, message: string) => {
       return entries.filter((entry) => entry.name.toLowerCase().includes(input.toLowerCase()));
     }
   });
-}
+};
+
+export const getLibrariesDetails = (tree: Tree) => Array.from(getProjects(tree))
```

Comment:
I offer to make a one function with a parameter `projectType` instead of  `getLibrariesDetails` and `getProjectsDetails`